src/main/java/com/java_essential/kopaiev/essential/homework/lesson1/task3/Book.java

```diff
@@ -0,0 +1,24 @@
+package com.java_essential.kopaiev.essential.homework.lesson1.task3;
+
+import java.awt.desktop.SystemSleepEvent;
+
+public class Book {
```

Comment:
The Book class must have placeholders for author, title, and content.
+
It should have a "show" method that prints the author, title, and content to the console.

Reply:
Change logic for Book class. Please check